assets/javascript/app-no.js

```diff
@@ -0,0 +1,93 @@
+$("#start").on("click", start);
+
+document.getElementById("start").innerHTML = 003 + ":" + 20;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The numeric literal '003' will have at different value at runtime.](https://app.codacy.com/app/columbia-fsf-f2019/trivia-game/pullRequest?prid=4555492)